src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexImporterTest.java

```diff
@@ -121,11 +121,4 @@ public void testGetDescription() {
                         + "The metadata is not required to be read here, as this class is NOT called at --import.",
                 importer.getDescription());
     }
-
-    @Test
-    public void testRecognizesDatabaseID() throws Exception {
```

Comment:
ok, in this case of course no special importer test is needed. sorry for the confusion 🍡